src/docs/orgs.md

```diff
@@ -0,0 +1,59 @@
+# Pants for Organisations
+
+Pants works well for organizations with a large number of developers working in a multi-tenant repo.
+Typically, repos becomes slower over time as more source and test files are added because most
+build systems build and test everything every time. Pants enables the benefits of a single repo
+with the speed of per-project repos by only operating on repo "slices".
+
+## Frontend & Backend Example
+
+Imagine a simple repo that contains both a frontend and backend service, as illustrated by the
+following dependency graph.
+
+![orgs](images/orgs.png)
+
+Consider a frontend developer that only works on the frontend service. With most build systems, the
+developer would need to compile everything every time, slowing them down over time as more and more
+code is added to repo. With Pants, the frontend developer only needs to compile the code directly
+used by their service. For example, consider building `org.pantsbuild.fe.service`. Regardless of
+how complex the backend code grows over time, the frontend build will not slow down.
+
+![orgs_fe](images/orgs_fe.png)
+
+Now consider a backend developer who only works on the backend service. When building the backend
+service, only the code required by the backend service is built, regardless of what other code
+is in the repo. The ability to build "slices" of the repo fundamental to enabling multiple users
+and teams to work together in a single repo. Note how only the backend code would be build when
+building `org.pantsbuild.be.service`:
+
+![orgs_be](images/orgs_be.png)
+
+## Continuous Integration
+
+Most organizations require test pass before merging a change to master. Pants enables fast CI by
+only running tests _affected_ by a change. That is, only targets that transitively depend on a
+changed target have their tests run. In practice, a change to `README.md` might not trigger *any*
+tests, a change to a leaf-node in the dependency graph would only trigger it's direct tests, and
+a change to a low-level node (e.g.: protobuf/thrift IDL file) might trigger most tests.
+
+For most users, the following script would work well for a pull-request builder type CI job:
+
+```bash
+#!/bin/bash
+
+set -x
+set -o
+set -e
+
+# Do not use the build cache when running tests to ensure no historical cruft
```

Comment:
This flag does not disable the buildcache: rather, it disables [zinc](https://github.com/sbt/zinc/#zinc) incremental compilation, which is/has-always-been error prone compared to clean builds.

In fact, by disabling incremental compilation, you actually _increase_ usage of the buildcache, because pants does not cache zinc incremental builds (for the reasons above), but it does cache clean builds.

Reply:
Thanks for the clarification, will update.

Do you have any thoughts on recommending `./pants test ::`, and how it pairs with `--test-junit-test-shard` as how to CI? This would be cleanest, though in practice I don't do that anymore because there were some code that didn't build together, which is it's own issue.

Reply:
> Do you have any thoughts on recommending ./pants test ::, and how it pairs with --test-junit-test-shard as how to CI?

In practice (when an organization becomes large enough) running `./pants test ::` puts too tight a constraint on which things are compatible.

Our goal with work on remoting is to make `./pants test ::` feasible again, and to do what you would expect (ie: enough ivy/coursier resolves to build all roots with a stable set of 3rdparty deps), but it at least 6-9 months out.